tests/stringnumber_tests.ts

```diff
@@ -275,8 +275,10 @@ function multi(options: TestOptions): void {
 
 function drawAccidentals(options: TestOptions): void {
   const f = VexFlowTests.makeFactory(options, 750);
-  const glyphScale = 39; // default font scale
-  const clefWidth = Glyph.getWidth('gClef', glyphScale); // widest clef
+  const el = new Element();
+  el.setText(String.fromCharCode(0xe050));
```

Comment:
@rvilarl  Why is this one still using the numeric code point? I'll merge, but can you fix this in the next PR?

Reply:
We should look for uses of fromCharCode() and fromCodePoint() and make sure they're really needed.